### PR TITLE
added rust starter code

### PR DESCRIPTION
**Pull request by Rujengelal**

### CODE REVIEW

File: rust/src/bid.rs [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+    pub challenger_id: String,
+    pub defender_bid: i64,
+    pub challenger_bid: i64,
+}
+
+pub fn get_bid(payload: &BidPayload) -> i32 {
+    if payload.bid_history.len() == 0 {
+        return MIN_BID;
+    }
+    PASS
```
Comment (pSapien) [inline rust/src/bid.rs:30]: @Rujengelal I'm unfamiliar with rust language, but pardon me if I'm wrong, would this return `PASS` without the `return` keyword?

**Comment by pSapien** [comment]
@Rujengelal Did you make sure that the endpoints are throwing valid responses? 

You could check it by playing through at least 5 games with `AllLocal` option enabled.

File: rust/src/bid.rs (hunk 1 above)
Comment (Rujengelal) [reply rust/src/bid.rs:30]: If it's last line of a function explicit `return` isn't required to return out of the function

**Comment by Rujengelal** [comment]
@pSapien  yes each endpoint returns valid responses. I've tested each end point from the sandbox as well as ran the sandbox in alllocal mode

**Comment by pSapien** [comment]
Thanks @Rujengelal  for the PR.

**Pull request merged**